Bitspace

keraliss opened the pull request:
These are the changed sections in bitspace page. also changed anmol's image




![Screenshot from 2024-06-25 12-31-28](https://github.com/Bitshala/website-v2/assets/145189844/edac2c08-f794-4196-915a-f673a5e46103)
![Screenshot from 2024-06-25 12-31-42](https://github.com/Bitshala/website-v2/assets/145189844/b05c2462-aae8-4abe-bda1-ab661cedfe14)
![Screenshot from 2024-06-25 12-31-54](https://github.com/Bitshala/website-v2/assets/145189844/166898a2-eac5-4aee-b93d-973fb21e454e)
![Screenshot from 2024-06-25 12-31-59](https://github.com/Bitshala/website-v2/assets/145189844/077926e5-a089-4c01-817c-7d4ba2f84156)
![Screenshot from 2024-06-25 12-32-17](https://github.com/Bitshala/website-v2/assets/145189844/3c53f08d-fce7-4ce8-9337-8bfd1c789c35)
![Screenshot from 2024-06-25 12-32-25](https://github.com/Bitshala/website-v2/assets/145189844/e952dc4c-81a7-4ee0-bf95-f13c69cc7ce6)
![Screenshot from 2024-06-25 12-32-31](https://github.com/Bitshala/website-v2/assets/145189844/4524beff-1af7-4db0-984f-36af748378e9)
![Screenshot from 2024-06-25 12-32-36](https://github.com/Bitshala/website-v2/assets/145189844/ad9925a5-0655-4655-be30-8c7fbe3ab6b4)

paperpsych commented:
The font for the "date" needs to be changed, use the image below as reference
![image](https://github.com/Bitshala/website-v2/assets/149538435/e5316381-55a1-4820-a150-4dbca3f84677)

In "See us in action" make it "**previous events**" instead of "previous meetups"

And the "Tell me more" button should take them to the meetup page for that link, or maybe the twitter post where all the highlights for the event are uploaded.

For the "Oops! We're still planning section" change the copy to "**Currently, we’re finalizing the details for the next events so come check this page again later or follow us on twitter to get updates.**" Make 'twitter' a hyperlink with the link to our twitter.

Rest of it seems okay !!

keraliss commented:
added event for memebook and fixed the nits

![Screenshot from 2024-07-01 12-40-00](https://github.com/Bitshala/website-v2/assets/145189844/f9af9473-3507-4be7-8acc-50966ad02017)
![Screenshot from 2024-07-01 12-39-51](https://github.com/Bitshala/website-v2/assets/145189844/06a668e4-79cf-42f1-a829-83e7d242bd71)

rajarshimaitra reviewed (approved):
ACK
